### PR TITLE
Add more tasks functionality

### PR DESCRIPTION
**Pull request by oxygen-consumer**
Please test it before merging, i'm too sleepy to do it :)

- Added status changes
- Added subtasks

### CODE REVIEW

File: TaskmanAPI/Controllers/ProjTasksController.cs [hunk 1]
```diff
@@ -65,4 +65,26 @@ public async Task<ActionResult<ProjTask>> AssignUsers(int id, string username)
...
+    [HttpDelete("{id}/remove_user/{username}")]
+    public async Task<ActionResult<ProjTask>> RemoveUsers(int id, string username)
+    {
+        var userId = await _userService.GetUserIdByUsername(username);
+        return Ok(await _projTasksService.RemoveUser(id, userId));
+    }
+
+    // api/Tasks/{id}/change_status/{status}
+    [HttpPut("{id}/change_status/{status}")]
+    public async Task<ActionResult<ProjTask>> ChangeStatus(int id, string status)
```
Comment (RaduIoana) [inline TaskmanAPI/Controllers/ProjTasksController.cs:79]: Changing status does not result in strings declared in Enum, it changes into something else (WaitingForActivation etc)

Comment (oxygen-consumer) [reply TaskmanAPI/Controllers/ProjTasksController.cs:79]: should be fixed in latest commit

File: TaskmanAPI/Controllers/ProjTasksController.cs [hunk 2]
```diff
@@ -65,4 +65,26 @@ public async Task<ActionResult<ProjTask>> AssignUsers(int id, string username)
         var userId = await _userService.GetUserIdByUsername(username);
         return Ok(await _projTasksService.AddUser(id, userId));
```
[the remark this answers is not in the document]
Comment (oxygen-consumer) [reply TaskmanAPI/Controllers/ProjTasksController.cs:66]: this should be fixed now too

**Review by RaduIoana — APPROVED** [review]
Is gud

**Comment by oxygen-consumer** [comment]
> Is gud

uwu

**Pull request merged**